vultr.gemspec

```diff
@@ -1,26 +1,26 @@
 # Run `rake vultr.gemspec` to update the gemspec.
 
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'vultr/version'
+require "vultr/version"
 
 Gem::Specification.new do |s|
-  s.name          = 'vultr'
-  s.version       = Vultr::VERSION::STRING
-  s.authors       = ['Gao Jiangmiao']
-  s.email         = ['tolbkni@gmail.com']
-  s.summary       = %q{Ruby bindings for VULTR API.}
-  s.description   = %q{Ruby bindings for VULTR API. VULTR APIs can be found here: https://www.vultr.com/api/}
-  s.homepage      = 'https://github.com/tolbkni/vultr.rb'
-  s.license       = 'MIT'
+  s.name = "vultr"
+  s.version = Vultr::VERSION
+  s.authors = ["Gao Jiangmiao"]
+  s.email = ["tolbkni@gmail.com"]
+  s.summary = "Ruby bindings for VULTR API."
+  s.description = "Ruby bindings for VULTR API. VULTR APIs can be found here: https://www.vultr.com/api/"
+  s.homepage = "https://github.com/tolbkni/vultr.rb"
+  s.license = "MIT"
 
-  s.files         = `git ls-files -z`.split("\x0")
-  s.executables   = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  s.test_files    = s.files.grep(%r{^(test|spec|features)/})
-  s.require_paths = ['lib']
+  s.files = `git ls-files -z`.split("\x0")
+  s.executables = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  s.test_files = s.files.grep(%r{^(test|spec|features)/})
+  s.require_paths = ["lib"]
 
-  s.required_ruby_version = '>= 1.9'
+  s.required_ruby_version = ">= 1.9"
```

Comment:
@excid3 Maybe we can bump the ruby version from ">= 1.9" to ">= 2.6", because the versions under 2.6 are EOL.

Reply:
Okay! Just checked and Faraday 1.7 requires Ruby >= 2.4, so we'll use that as our minimum version.

Faraday 2.0 is just around the corner and requires Ruby >= 2.6. I don't see a need to use that yet.